### ISSUE TITLE
Coding : Develop Introduction section of Home Page

### ISSUE THREAD

**Issue by vj-codes** [anitab-org/anitab-org.github.io#109]
### Is your feature request related to a problem? Please describe.
As a developer,
I need the following section to be developed so that I can add is as a Home component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components)

### Mocks
Develop the section in red box only.

![Screenshot (27)_LI](https://user-images.githubusercontent.com/60894542/88478134-703d5100-cf63-11ea-917a-1398ab889d03.jpg)

Font Size for heading EVENTS: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

[Design link for development](https://xd.adobe.com/view/09a9fdf0-a193-4b3f-47c2-216055c7ca42-7f98/)

### Update [Required]
- [ ] Should follow all the margin guidelines mentioned above and should be responsive

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6-8 hours

**Comment by nandini45** [comment]
@anitab-org/coding-team can  anyone take up this issue?

**Comment by Sonal240** [comment]
@vj-codes the margins are incorrect

**Comment by vj-codes** [comment]
@Sonal240 which ones? I checked with the doc didn't find any

**Comment by Sonal240** [comment]
It's not 16...it's 32 in total see the doc. I have not designed a separate component or so, hence I didn't break 32pt margin to 16-16...also 32pt margin from horizontal line

**Comment by saideepesh000** [comment]
@Sonal240 , @nandini45, @vj-codes I would like to work on it.

**Comment by sakshi1499** [comment]
@saideepesh000 Its on hold right now, we'll get back to you once its confirmed.

**Comment by nandini45** [comment]
@vj-codes  did you update the design metrics?

**Comment by vj-codes** [comment]
@nandini45 @Sonal240 please chack once!

**Comment by Sonal240** [comment]
The screenshot is not visible @vj-codes

**Comment by vj-codes** [comment]
> The screenshot is not visible @vj-codes 

It is visible,can you try refreshing? I'm on phone right it's visible on it too![Screenshot_2020-08-17-11-28-19-825_com.github.android.jpg](https://user-images.githubusercontent.com/60894542/90361896-ee7da680-e07c-11ea-9f0b-0d6c2ec5065b.jpg)

**Comment by Sonal240** [comment]
it's not I tried it refreshing it as well

**Comment by Sonal240** [comment]
on clicking the link it is though...so I don't think it would create an issue...

**Comment by adithyaakrishna** [comment]
Can I work on this issue? :)

**Comment by nandini45** [comment]
@adithyaakrishna  any updates?

**Comment by khushishikhu** [comment]
may i work on this issue

**Comment by Sonal240** [comment]
@khushishikhu this issue is assigned to someone else may you please look for some other issue thanks 😊

**Comment by adithyaakrishna** [comment]
Hi @nandini45 @Sonal240 , Sorry for not giving updates here, but I'm currently held up with my final year project work, you can assign it to @khushishikhu

**Comment by nandini45** [comment]
ok

**Comment by pallavithakare28** [comment]
@vj-codes @nandini45 May I work on this issue?

**Comment by vj-codes** [comment]
Assigning you @pallavithakare28 
Happy coding!

**Comment by pallavithakare28** [comment]
@vj-codes Thank you :)

**Comment by Aaishpra** [comment]
Hey, @vj-codes if @pallavithakare28 somehow couldn't complete the work here . I would like to get assigned to this as i have already worked on a similar kind of issue .😊

**Comment by pallavithakare28** [comment]
> Hey, @vj-codes if @pallavithakare28 somehow couldn't complete the work here. I would like to get assigned to this as I have already worked on a similar kind of issue.😊

@Aaishpra @vj-codes I am working on this issue currently. As I am new to open source and GitHub too, so it is taking the time. I'll try to give my best to solve this issue. And soon will make a PR.  It'll be great if you can guide me where I have to start. Thank you :)

**Comment by vj-codes** [comment]
@pallavithakare28 sure, take your time but also remember to update your progress after every 3 days and if you come into any errors or doubts you can ask on zulip or here in comments:)

**Comment by Aaishpra** [comment]
> > Hey, @vj-codes if @pallavithakare28 somehow couldn't complete the work here. I would like to get assigned to this as I have already worked on a similar kind of issue.😊
> 
> @Aaishpra @vj-codes I am working on this issue currently. As I am new to open source and GitHub too, so it is taking the time. I'll try to give my best to solve this issue. And soon will make a PR.  It'll be great if you can guide me where I have to start. Thank you :)

Yes, ask any question would be more than delighted to help u through this.

**Comment by pallavithakare28** [comment]
@vj-codes @Aaishpra  Can you please help me? I am not getting from where I have to start. And from where I have to make changes.

**Comment by Aaishpra** [comment]
Hey, @pallavithakare28 you can start by making a new folder or a new file inside the src/Components folder. The next part would be to go to the design link (given in the issue description) and download the image and understand all the CSS required for designing this section. After that, you can probably start writing the code in the new file you created, my suggestion would be to have a look at the code for already developed section before writing your own new code. Hope, this helps.

**Comment by pallavithakare28** [comment]
Hey, @Aaishpra Got it.  Thank you for helping me.

**Comment by pallavithakare28** [comment]
@vj-codes I am working on it. I hope you consider it.

**Comment by nandini45** [comment]
@pallavithakare28  there has been no updates from you.
if are working then update us with what you have done and the issue you are facing to complete it?

**Comment by Aaishpra** [comment]
@nandini45 well if this issue gets available for reassignment i am more than happy to work on it.

**Comment by pallavithakare28** [comment]
@nandini45 Sorry for delay, but I am busy with exam so can't make PR now. It's ok for me if someone wants to work on this issue. Sorry for the inconvenience.

**Comment by Harshita-Kanal** [comment]
Is it available,  can I take this up?

**Comment by Akanksha1212** [comment]
Yes @Harshita-Kanal you can take this up. Let us know if you need any help.

**Comment by nandini45** [comment]
@Harshita-Kanal  are you working on it.? if yes
please mention the updates

**Comment by tichnas** [comment]
Please assign me this if no update comes from @Harshita-Kanal in a few days.

**Comment by Harshita-Kanal** [comment]
Hello, sorry for not giving updates here, I had my exams so couldn't complete. You can unassign me and I am very  sorry for the delay and inconvenience

**Comment by tichnas** [comment]
@Harshita-Kanal If you're still interested, I think we'll be happy to wait for you :)

**Comment by nandini45** [comment]
@tichnas here... do give us the updates of your work after 3 4 days.. if any issue or doubts feel free to ping us here🙂

**Comment by tichnas** [comment]
@nandini45 Till now I've designed this much:
![image](https://user-images.githubusercontent.com/56799401/100220457-68a3ae00-2f3d-11eb-87b8-cc70a5123ab4.png)
Please let me know any improvements. Also, are the icons acting as a link to Medium, Zulip and Github?

**Comment by nandini45** [comment]
Yes they are the links to the community channels..

**Comment by nandini45** [comment]
It looks great Just one thing can u check the right side alignments??

**Comment by nandini45** [comment]
If you think 32 it a bit too much space make it 16 both sides.

**Comment by nandini45** [comment]
![20201125_165516](https://user-images.githubusercontent.com/62046625/100221725-15caf600-2f3f-11eb-9f2e-6263a610b7ac.png)
I am taking about these alignments

**Comment by purnima143** [comment]
@nandini45  @tichnas I wanna give small advice regarding the homepage design. The current state of the homepage is full with texts only, that looks bulky.

-  Keep the page **minimalistic and free of clutter**. In someplace instead of text, you can use more images and icons to present your thought. 

- “**Readability**” measures how easy it is for people to recognize words, sentences, and phrases.

- Create one contrasting landing page, that will change the entire look. 

I hope my advice will be helpful for you guys to design the beautiful homepage.🙂

**Comment by tichnas** [comment]
@nandini45 @vj-codes What changes do you suggest considering @purnima143's advice?

**Comment by nandini45** [comment]
@tichnas  yes agreed but right now i can't ask u  to redesign it again.. there are many features which we will need to work on. I will  say do what was initially said , if u can make minimal changes without affecting the proposed design.. well and good..make changes but dont change the layouts

**Comment by tichnas** [comment]
@nandini45 No issues.
To be sure that I did what was expected, can you please see the final screen-shot before opening a PR.

![image](https://user-images.githubusercontent.com/56799401/101831278-b7675f80-3b5b-11eb-9848-1df7e8295eb9.png)

**Comment by nandini45** [comment]
@tichnas  it looks great..please go ahead with the PR

**Comment by annabauza** [comment]
@tichnas I release this issue due to lack of respond.

**Comment by prachi237** [comment]
@annabauza ma'am so basically we need to code the front-end of the home page right, like the design that have been proposed above

**Comment by nandini45** [comment]
@prachi237 the design has been finalized . you just have to code it.
if you can i will assign you the issue

**Comment by prachi237** [comment]
@nandini45  yes ma'am I would like to work on it, but can u plz send be the UI details and tech stacks to be used

**Comment by nandini45** [comment]
@prachi237  no ma'am please just nandini.
tech stack is angular JS 
ui details see this our home page here a small section is not there 
 you have to develop that only 
![image](https://user-images.githubusercontent.com/62046625/109157221-c1325380-7797-11eb-8526-5af11b63b361.png)

this is what you have to add
![image](https://user-images.githubusercontent.com/62046625/109157298-d9a26e00-7797-11eb-948f-2cf9188f945f.png)

although if u want to contribute more to designing specifically you can join our design team on zulip

**Comment by vj-codes** [comment]
@nandini45 I think it's react js not angular js for tech stack , do confirm .

**Comment by Aaishpra** [comment]
@nandini45 @vj-codes actually react native is used  here😉 i am sure of it

**Comment by vj-codes** [comment]
@Aaishpra yes earlier we used native now we are moving to react js 
(check comment by Anna [here](https://github.com/anitab-org/anitab-org.github.io/issues/42#issuecomment-780130118) )

**Comment by Aaishpra** [comment]
> @Aaishpra yes earlier we used native now we are moving to react js 
> (check comment by Anna [here](https://github.com/anitab-org/anitab-org.github.io/issues/42#issuecomment-780130118) ) 

Ohh really thanks for informing @vj-codes

**Comment by nandini45** [comment]
I am so sorry my mistake got a but confused

**Comment by nandini45** [comment]
@prachi237  are you working on it? 
if not i will un-assign  you

**Comment by sanchibansal340** [comment]
@nandini45 From what I understand the code has now been converted to ReactJS, right?
If yes, can I please try to work on this issue, if available?

**Comment by nandini45** [comment]
@sanchibansal340  sure, i assigned the issue to you

**Comment by sanchibansal340** [comment]
Hi @nandini45, I am done with the issue. Below are the changes that I have made, let me know the changes. From the previous PR, it looks like Lorem Ipsum text isn't accepted, any updates about the actual content? 
![Screenshot (172)](https://user-images.githubusercontent.com/51092036/124281227-24c5f680-db67-11eb-9f7f-bf45605774d7.png)

**Comment by sanchibansal340** [comment]
> Hi @nandini45, I am done with the issue. Below are the changes that I have made, let me know the changes. From the previous PR, it looks like Lorem Ipsum text isn't accepted, any updates about the actual content?
> ![Screenshot (172)](https://user-images.githubusercontent.com/51092036/124281227-24c5f680-db67-11eb-9f7f-bf45605774d7.png)

@vj-codes Can you please look into this?

**Comment by vj-codes** [comment]
@sanchibansal340 the intro part that I see when logging in on zulip might be good except for the last 2 lines in the screenshot below.
@isabelcosta @annabauza what say?

![Screenshot (59)](https://user-images.githubusercontent.com/60894542/127508180-e7356122-6a21-407f-b1c0-ba15cd7bf76f.png)

**Comment by sanchibansal340** [comment]
@vj-codes hey I totally forgot about this issue. Shall I move forward with this content then?

**Comment by nandini45** [comment]
@sanchibansal340  i agree with @vj-codes  . update it and submit the PR .
we will review it 
thanks

**Comment by mudrikapandey** [comment]
Is this issue is solved ??
